catalog/app/containers/Profile/messages.js

```diff
@@ -34,6 +38,26 @@ export default defineMessages({
     id: 'app.containers.Profile.learnMore',
     defaultMessage: 'Compare plans',
   },
+  membersAdd: {
+    id: 'app.containers.Profile.membersAdd',
+    defaultMessage: 'Add member',
+  },
+  membersRead: {
```

Comment:
this and the next four strings are duplicated in another file. why?

Reply:
seems like they were initially put here and then copied to `Admin/messages` and forgotten. i'll clean this up